packages/main/src/Menu.ts

```diff
@@ -500,6 +500,10 @@ class Menu extends UI5Element {
 	}
 
 	_getSubmenuReference(item: MenuItem) {
+		if (!item) {
```

Comment:
By definition this method doesn't accept nullish values. 

It seems that the definition is not defined correctly or the method is not used correctly somewhere.